packages/usa-character-count/src/test/character-count.spec.js

```diff
@@ -139,12 +157,39 @@ tests.forEach(({ name, selector: containerSelector }) => {
       EVENTS.input(input);
 
       assert.strictEqual(input.validationMessage, VALIDATION_MESSAGE);
+      assert.strictEqual(input.classList.contains(INPUT_ERROR_CLASS), true);
       assert.strictEqual(
         statusMessageVisual.classList.contains(MESSAGE_INVALID_CLASS),
         true
       );
     });
 
+    it("should add form group error classes on focusout when the input has error class", () => {
+      input.classList.add(INPUT_ERROR_CLASS);
+
+      EVENTS.focusout(input);
+
+      assert.strictEqual(
+        formGroup.classList.contains(FORM_GROUP_ERROR_CLASS),
+        true
+      );
+      assert.strictEqual(label.classList.contains(LABEL_ERROR_CLASS), true);
+    });
+
+    it("should remove form group error classes on focusout when the input does not have error class", () => {
+      assert.strictEqual(input.classList.contains(INPUT_ERROR_CLASS), false);
+      formGroup.classList.add(FORM_GROUP_ERROR_CLASS);
+      label.classList.add(LABEL_ERROR_CLASS);
+
+      EVENTS.focusout(input);
+
+      assert.strictEqual(
+        formGroup.classList.contains(FORM_GROUP_ERROR_CLASS),
+        false
+      );
+      assert.strictEqual(label.classList.contains(LABEL_ERROR_CLASS), false);
+    });
```

Comment:
> [!NOTE]
> New testing requirements to check for the addition / removal of error states on input and focusout

Reply:
Removed in [b7fa1f0](https://github.com/uswds/uswds/pull/5908/commits/b7fa1f0494c815f2c04032cf025d16944ea6912c)